### PR TITLE
prepare for translation

### PR DESCRIPTION
**Pull request by kaibolay**
- removed unused strings
- extracted hard-coded strings
- added `translation_description`

### CODE REVIEW

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -13,40 +13,37 @@
 <!-- limitations under the License. -->
 
 <resources>
-  <string name="signin_dialog_title">Enable testing features</string>
-  <string name="singin_dialog_message">Enable testing features like new build alerts and in-app feedback.</string>
-  <string name="singin_yes_button">Turn on</string>
-  <string name="singin_no_button">Not now</string>
-  <string name="no_update_available">No new release available, try calling checkForUpdate.</string>
-  <string name="update_dialog_title">New Version Available</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:21]: lowercase "version available"

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="download_pending">Download pending</string>
-  <string name="install_failed">Install failed</string>
-  <string name="install_canceled">Install canceled</string>
-  <string name="update_canceled">Update canceled </string>
-  <string name="notifications_group_name">Firebase App Distribution</string>
-  <string name="app_update_notification_channel_name">App updates</string>
-  <string name="app_update_notification_channel_description">Shows progress of in-app updates</string>
-  <string name="feedback_launch_failed">Failed to launch feedback</string>
-  <string name="feedback_unidentified_release">Release not found. This app may not have been installed by App Distribution, or you may no longer have access.</string>
-  <string name="feedback_no_release">Would have sent feedback, but did not due to development mode.</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:36]: Is this due to the Developer Mode issue? Is there a way to rephrase this such that the user knows what happened and how to resolve the issue?

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="feedback_notification_channel_name">Feedback</string>
-  <string name="feedback_notification_channel_description">Tap to leave feedback</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:38]: Recommend replacing "leave" with "enter"

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="feedback_notification_title">We want your feedback!</string>
-  <string name="feedback_notification_text">Tap to leave feedback for %1$s</string>
-  <string name="unknown_sources_dialog_title">Enable Unknown Sources</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:41]: Use sentence case

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="unknown_sources_dialog_description">To install the update enable unknown sources</string>
-  <string name="unknown_sources_yes_button">Settings</string>
-  <string name="back_button_description">Back</string>
-  <string name="send_feedback">Send feedback</string>
-  <string name="send_button_description">Send</string>
-  <string name="sender">From: %1$s</string>
-  <string name="feedback_text_hint">Have feedback? Let us know how we can make this app better.</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:48]: Replace with "Have feedback? Tell us how we can improve this app."

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="signin_dialog_title">Enable testing features</string>
-  <string name="singin_dialog_message">Enable testing features like new build alerts and in-app feedback.</string>
-  <string name="singin_yes_button">Turn on</string>
-  <string name="singin_no_button">Not now</string>
-  <string name="no_update_available">No new release available, try calling checkForUpdate.</string>
-  <string name="update_dialog_title">New Version Available</string>
-  <string name="update_yes_button">Update</string>
-  <string name="update_no_button">Cancel</string>
-  <string name="downloading_app_update">Downloading in-app update…</string>
-  <string name="download_completed">Download completed</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:25]: Replace "completed" with "complete"

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="app_update_notification_channel_description">Shows progress of in-app updates</string>
-  <string name="feedback_launch_failed">Failed to launch feedback</string>
-  <string name="feedback_unidentified_release">Release not found. This app may not have been installed by App Distribution, or you may no longer have access.</string>
-  <string name="feedback_no_release">Would have sent feedback, but did not due to development mode.</string>
-  <string name="feedback_notification_channel_name">Feedback</string>
-  <string name="feedback_notification_channel_description">Tap to leave feedback</string>
-  <string name="feedback_notification_title">We want your feedback!</string>
-  <string name="feedback_notification_text">Tap to leave feedback for %1$s</string>
-  <string name="unknown_sources_dialog_title">Enable Unknown Sources</string>
-  <string name="unknown_sources_dialog_description">To install the update enable unknown sources</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:42]: Add a comma after "To install the update"

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 8]
```diff
@@ -13,40 +13,37 @@
 <!-- limitations under the License. -->
 
 <resources>
-  <string name="signin_dialog_title">Enable testing features</string>
-  <string name="singin_dialog_message">Enable testing features like new build alerts and in-app feedback.</string>
-  <string name="singin_yes_button">Turn on</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:18]: Why not "Enable"?

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="singin_no_button">Not now</string>
```
Comment (VictoriaAuto) [inline firebase-appdistribution/src/main/res/values/strings.xml:19]: Would "Cancel" be more appropriate here, if it's canceling an action?

**Review by VictoriaAuto — CHANGES_REQUESTED** [review]
Hi Kai, I reviewed the strings and left some comments. Let me know if you have any questions about my feedback. Thank you! :-)

File: firebase-appdistribution/src/main/res/values/strings.xml (hunk 9 above)
Comment (lfkellogg) [reply firebase-appdistribution/src/main/res/values/strings.xml:19]: The action (trying to enable the SDK) was likely not taken by the user in this case. It is typically triggered by the developer when the app is first opened.

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -13,40 +13,37 @@
...
-  <string name="unknown_sources_yes_button">Settings</string>
-  <string name="back_button_description">Back</string>
-  <string name="send_feedback">Send feedback</string>
-  <string name="send_button_description">Send</string>
-  <string name="sender">From: %1$s</string>
-  <string name="feedback_text_hint">Have feedback? Let us know how we can make this app better.</string>
-  <string name="screenshot">Screenshot</string>
-  <string name="no_screenshot">Screenshot (not available)</string>
-  <string name="screenshot_image_description">Screenshot image</string>
+  <string name="signin_dialog_title" translation_description="Title of the dialog encouraging the user to enable Firebase App-Distribution features">Enable testing features</string>
```
Comment (lfkellogg) [inline firebase-appdistribution/src/main/res/values/strings.xml:16]: Nit: don't hyphenate App Distribution :)

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -13,40 +13,37 @@
...
+  <string name="feedback_unidentified_release" translation_description="Toast explaining feedback can't be provided, because release can't be found">Release not found. This app may not have been installed by App Distribution, or you may no longer have access.</string>
+  <string name="feedback_no_release" translation_description="Toast explaining that feedback would have been sent, but was not, since development mode was active">Would have sent feedback, but did not due to development mode.</string>
+  <string name="feedback_notification_channel_name" translation_description="Name of the notification channel for in-app feedback">Feedback</string>
+  <string name="feedback_notification_channel_description" translation_description="Description of the notification channel for in-app feedback">Shows notification to leave feedback about app</string>
+  <string name="feedback_notification_title" translation_description="Title of notification encouraging testers to leave feedback">We want your feedback!</string>
+  <string name="feedback_notification_text" translation_description="Text of notification encouraging testers to leave feedback">Tap to leave feedback for %1$s</string>
```
Comment (lfkellogg) [inline firebase-appdistribution/src/main/res/values/strings.xml:38]: Nit: how about "... to leave feedback **for the app**" just so it's clear what the placeholder represents.

File: firebase-appdistribution/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -13,40 +13,37 @@
...
+  <string name="unknown_sources_dialog_title" translation_description="Title of dialog describing that installation from unknown sources needs to be enabled">Enable Unknown Sources</string>
+  <string name="unknown_sources_dialog_description" translation_description="Text of dialog describing why installation from unknown sources needs to be enabled">To install the update enable unknown sources</string>
+  <string name="unknown_sources_yes_button" translation_description="Label on button launching settings to enable installation from unknown sources">Settings</string>
+  <string name="back_button_description" translation_description="Description for image button (back) which aborts feedback">Back</string>
+  <string name="send_feedback" translation_description="Text next to send button which sends feedback">Send feedback</string>
+  <string name="send_button_description" translation_description="Description of image button (send) which sends feedback">Send</string>
+  <string name="feexdback_text_hint" translation_description="">Have feedback? Let us know how we can make this app better.</string>
```
Comment (lfkellogg) [inline firebase-appdistribution/src/main/res/values/strings.xml:45]: The last 4 need `translation_description`s

**Review by lfkellogg — APPROVED** [review]

**Pull request merged**